tests/integration/conftest.py

```diff
@@ -33,6 +34,68 @@
 NODEBALANCER_BASE_CMD = ["linode-cli", "nodebalancers"]
 
 
+@pytest.fixture(autouse=True, scope="session")
+def linode_cloud_firewall():
+    def get_public_ip(ip_version="ipv4"):
+        url = (
+            f"https://api64.ipify.org?format=json"
+            if ip_version == "ipv6"
+            else f"https://api.ipify.org?format=json"
+        )
+        response = requests.get(url)
+        return str(response.json()["ip"])
+
+    def create_inbound_rule(ipv4_address, ipv6_address):
+        rule = [
+            {
+                "protocol": "TCP",
+                "ports": "22",
+                "addresses": {
+                    "ipv4": [f"{ipv4_address}/32"],
+                    "ipv6": [f"{ipv6_address}/128"],
+                },
+                "action": "ACCEPT",
+            }
+        ]
+        return json.dumps(rule, indent=4)
+
+    # Fetch the public IP addresses
+    ipv4_address = get_public_ip("ipv4")
+    ipv6_address = get_public_ip("ipv6")
+
+    inbound_rule = create_inbound_rule(ipv4_address, ipv6_address)
```

Comment:
Firewall will automatically pick up ipv4 and ipv6 of machine under test and add it as `inbound_rule`